modules/appeals_api/.bruno_appeals_api/helpers.js

```diff
@@ -0,0 +1,85 @@
+const uuid = require('uuid')
```

Comment:
This JS file replicates the script provided by the Pivot team for Postman, [which is here](https://github.com/department-of-veterans-affairs/vets-api-clients/tree/master/samples/oauth_ccg_postman_pre_request_script). I adjusted it a little bit to work with the built-in libraries that Bruno ships (includes uuid and axios), and to manage the `jsrsasign` dependency using `npm` instead of an ad-hoc download script. I'll include instructions about this in the docs for these example requests.